packages/neuron-wallet/src/cell.ts

```diff
@@ -6,6 +6,22 @@ export interface OutPoint {
   index: number
 }
 
+interface TypeScript {
+  version: number
+  args: string[]
```

Comment:
in node we prefer to use Buffer or Uint8Array

Reply:
Node use uint8array but RPC use hex string as input and output, I don't know why we should use uint8array

Reply:
I strongly recommend that we use `string` for CKB internal `[uint8]`, and `string[]` for `[[uint8]]` (`vec<vec<uint8>>`), as long as RPC returns hex string. Only use bytes when hashing internally.